frontend/src/components/table/TableItem.js

```diff
@@ -212,9 +220,9 @@ class TableItem extends PureComponent {
    * @param {object} [item] - widget data object
    * @param {boolean} [select] - flag if selected cell should be cleared
    */
-  _editProperty = (e, property, focus, item, select) => {
+  _editProperty = ({ e, property, focus, item, select, mark }) => {
     if (item ? !item.readonly : true) {
-      if (this.state.edited === property && e) e.stopPropagation();
+      if (this.state.edited === property && e) e.persist(); // e.stopPropagation();
```

Comment:
Why do we need to persist the event ? Are we using it in any way and need to access it past this point ?

Reply:
<img width="1669" alt="Screenshot 2020-08-17 at 16 58 36" src="https://user-images.githubusercontent.com/1708561/90404374-fe13e400-e0aa-11ea-84ae-5eaa097d3362.png">

It is being use when you get the focus on child components. If you are to use e.stopPropagation while editing you will get a bunch of warnings as in the screenshot. This is the reason I removed it and persisted the event.